packages/govuk-frontend/tasks/watch.mjs

```diff
@@ -18,7 +20,7 @@ export const watch = (options) => gulp.parallel(
     gulp.watch([
       `${slash(options.srcPath)}/govuk/**/*.scss`,
       `!${slash(options.srcPath)}/govuk/vendor/*`
-    ], npm.script('lint:scss', options))
+    ], npm.script('lint:scss:cli', [join(options.workspace, '**/*.scss')]))
```

Comment:
**question** Does this need `slash`ing (or use of `node:path/posix`) for Windows environments, as `join` will use the OS separator I believe?

Reply:
The arguments get passed straight into [`@npmcli/run-script`](https://www.npmjs.com/package/@npmcli/run-script) so I suppose the question is:

**Will ESLint and Stylelint accept OS-native slashes via CLI?**
I've just assumed they would do and not done any conversion

Have you got a Windows machine to hand??

Reply:
No Windows machine at hand unfortunately (would need to install a few things on the modern.ie VMs, which I haven't done). I was asking because we `slash` most of the other glob we pass around (like L21-22 right above) so figured we may need to do so there as well. 

For safety, would you mind adding a `slash` around it, please?

Reply:
You'd think CLI tools on Windows would rather accept Windows paths than POSIX ones?

I've got a Windows 10 partition so I'll boot into it tomorrow and check

Reply:
✅ ESLint supports Windows paths on Windows
🚫 Stylelint does not support Windows paths on Windows

I've pushed up a [commit with `slash()`](https://github.com/alphagov/govuk-frontend/pull/3621/commits/6ba66c96e6e86a35c957cd996b4e2e6c0bd47fd1) 🤦‍♂️